pkg/plugin/admissioncontrollers/webhook/admission_controller_test.go

```diff
@@ -232,3 +258,26 @@ func TestValidateConfigMap(t *testing.T) {
 	assert.Equal(t, err.Error(), "required config 'queues.yaml' not found in this configmap")
 	// skip further validations which depends on the webservice of yunikorn-core
 }
+
+func TestGenerateAppID(t *testing.T) {
+	appID := generateAppID("this-is-a-pod-name")
+	t.Logf("generated appID %s", appID)
```

Comment:
These log messages show up as failures in the github runs. We don't need them to be logged for every test run, i'll remove on commit.